src/Syllable.php

```diff
@@ -754,13 +754,28 @@ private function parseWord($word)
             return [$word];
         }
 
+        $wordLowerCased = mb_strtolower($word);
+
         // Is it a pre-hyphenated word?
-        if (isset($this->hyphenation[$word])) {
-            return mb_split('-', $this->hyphenation[$word]);
+        if (isset($this->hyphenation[$wordLowerCased])) {
```

Comment:
Maybe refactor the splitting algorithm into a new method for readability?

Reply:
Done.